### PR TITLE
GitVersion: added version 5.11.1

### PR DESCRIPTION
**Pull request by arturcic**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90742)

### CODE REVIEW

**Comment by arturcic** [comment]
For some reason https://github.com/microsoft/winget-pkgs/pull/88540 was merged but no changes are merged

**Comment by russellbanks** [comment]
@msftbot zip

**Review by ItzLevvie — DISMISSED** [review]

**Comment by Trenly** [comment]
[Policy] Reset Feedback

File: manifests/g/GitTools/GitVersion/5.11.1/GitTools.GitVersion.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS2.CRLF.7-3-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: GitTools.GitVersion
+PackageVersion: 5.11.1
+InstallerType: zip
+Commands:
+- gitversion
+ReleaseDate: 2022-11-08
+Installers:
```
Comment (stephengillie) [inline manifests/g/GitTools/GitVersion/5.11.1/GitTools.GitVersion.installer.yaml:10]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Git.Git
Installers:
```

Comment (stephengillie) [reply manifests/g/GitTools/GitVersion/5.11.1/GitTools.GitVersion.installer.yaml:10]: Hi @arturcic, 

This package appears to depend on Git to operate. When launched, it gives this error:
> Cannot find the .git directory

**Comment by arturcic** [comment]
@stephengillie anything i need to do to fix this PR and get it going?

File: manifests/g/GitTools/GitVersion/5.11.1/GitTools.GitVersion.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+PackageIdentifier: GitTools.GitVersion
+PackageVersion: 5.11.1
+InstallerType: zip
+Commands:
+- gitversion
+ReleaseDate: 2022-11-08
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Git.Git
+Installers:
```
Comment (stephengillie) [inline manifests/g/GitTools/GitVersion/5.11.1/GitTools.GitVersion.installer.yaml:13]:
```suggestion
    - PackageIdentifier: Git.Git
Installers:
```

Comment (stephengillie) [reply manifests/g/GitTools/GitVersion/5.11.1/GitTools.GitVersion.installer.yaml:13]: It appears that I added an incorrect line ending, or other invisible character that's blocking dependencies. I believe that I have removed it with this suggestion.

**Comment by stephengillie** [comment]
Hi @arturcic,

> @stephengillie anything i need to do to fix this PR and get it going?

Sorry for the delay - we're still catching up from some issues that impacted us last week. 
* https://github.com/microsoft/winget-pkgs/issues/95543

This was compounded by my adding a bad line ending or other invisible character, which blocked dependency install. Hopefully this PR should be good to go with the most recent suggestion.

File: manifests/g/GitTools/GitVersion/5.11.1/GitTools.GitVersion.installer.yaml (hunk 2 above)
Comment (arturcic) [reply manifests/g/GitTools/GitVersion/5.11.1/GitTools.GitVersion.installer.yaml:13]: It seems the Commit suggestion for some reason is disabled, so what line ending should be? I will apply it myself and push again

**Review by arturcic — COMMENTED** [review]
LGTM

**Comment by stephengillie** [comment]
@arturcic,

Apologies, I mistakenly thought we had added dependency support.

**Comment by arturcic** [comment]
> @arturcic,
> 
> Apologies, I mistakenly thought we had added dependency support.

Then what is the option? To wait version 1.5?

**Comment by arturcic** [comment]
I wonder how this PR was force-pushed, from my commit, to a different commit not related to my PR at all
@[pull](https://github.com/apps/pull) pull bot [force-pushed](https://github.com/microsoft/winget-pkgs/compare/82f4e35a892d999979d151d54416fb206b9ff98a..29815bfc4d644ca5404700cd5b38298969b9342e) the master branch from [82f4e35 ](https://github.com/microsoft/winget-pkgs/commit/82f4e35a892d999979d151d54416fb206b9ff98a)to [29815bf ](https://github.com/microsoft/winget-pkgs/commit/29815bfc4d644ca5404700cd5b38298969b9342e)[11 minutes ago](https://github.com/microsoft/winget-pkgs/pull/90742#event-8596249328)

**Comment by arturcic** [comment]
@stephengillie @Madhusudhan-MSFT  - that is already second time my PRs are reset to different commits that are not related to my PR (a different manifest not GitVersion). Seems like the pull bot does some weird work

**Comment by arturcic** [comment]
I filled in a 3rd attempt - #97653

**Comment by stephengillie** [comment]
> @stephengillie @Madhusudhan-MSFT - that is already second time my PRs are reset to different commits that are not related to my PR (a different manifest not GitVersion). Seems like the pull bot does some weird work

@denelon

**Comment by arturcic** [comment]
Thanks you for the information I will have a look

**Pull request merged**